prodactivity-frontend/managers/AuthenticationManager.tsx

```diff
@@ -4,29 +4,56 @@ import KeychainHelper from '../helpers/KeychainHelper';
 import { TokenType } from '../application/Enums';
 
 export default class AuthenticationManager {
-
     baseURL = 'http://localhost:5000/authentication';
 
-    constructor() {
-    }
+    constructor() {}
 
     /**
      * @param {string} email The user's email address for sign in
      * @param {string} password The user's password for sign in
-     * 
+     *
      * @returns {AuthenticationViewModel} View Model containing all info for the login
      */
     async signIn(email: string, password: string): Promise<Result<AuthenticationViewModel>> {
         try {
-            let response = await FetchHelper.post(`${this.baseURL}/signIn`, {"email": email, "password": password});
+            let response = await FetchHelper.post(`${this.baseURL}/signIn`, { email: email, password: password });
+            if (response) {
```

Comment:
Why the null check on these results? Is the backend returning null in case of errors?

Reply:
FetchHelper can return undefined so that's why it is being checked, but in the issue I'm working on now that's changed so I'll remove the null check.

Reply:
Yeah after I wrote that comment I remembered I was the one who made FetchHelper's return type possibly undefined. I think I did that because I wasn't sure what the response body .json() method would return if we had a non-json body.